.github/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 3. Bug - H4 Mislabelled as H - thanks to @luke11brown (Luke Brown)
 4. AIRAC (1805) - Tango routes to RNAV - thanks to @daleb99 (Dale Brooksby)
 5. AIRAC (1805) - Lower airways converted to RNAV - thanks to @daleb99 (Dale Brooksby)
+6. AIRAC (1805) - Blackbushe (EGLK) SMR added - thanks to @epicdemon99 (Nathaniel Leff)
```

Comment:
Nitpicking, but this should read:
Enhancement - Blackbushe (EGLK) SMR added - thanks to @epicdemon99 (Nathaniel Leff)